.changeset/ninety-hats-serve.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend-module-azure': patch
+---
+
+This will add the ability to use Azure DevOps with multi project with a single value which is a new feature as previously this had to be done manually for each project that you wanted to add
```

Comment:
Would be great to have an example here or a link to the docs. These end up as part of the CHANGELOG and release notes so for those upgrading it's super helpful to have